server/models.py

```diff
@@ -93,39 +164,46 @@ class SeatAssignment(db.Model):
     seat_id = db.Column(db.ForeignKey('seats.id'), index=True, nullable=False)
     emailed = db.Column(db.Boolean, default=False, index=True, nullable=False)
 
-    student = db.relationship('Student', backref=backref('assignment', uselist=False))
-    seat = db.relationship('Seat', backref=backref('assignment', uselist=False))
 
 def slug(display_name):
     return re.sub(r'[^A-Za-z0-9._-]', '', display_name.lower())
 
 # Flask-CLI commands
 # Run with `flask <cmd>`
+
+
 @app.cli.command('initdb')
```

Comment:
This stuff feels like it belongs in a different file...

Reply:
yea sure 

these are like rails scripts or npm actions.

if we use poetry we could define them in poetry file. Even if not we can separate them out to a differnt file too. There are a bunch of flask cli commands defined for running tests too. Should consolidated them.

Reply:
Yeah, the flask API makes sense. I mean, a different file is useful just to make them findable.